src/Services/RabbitMQService.cs

```diff
@@ -88,9 +95,30 @@ internal static ConnectionFactory GetConnectionFactory(string connectionString,
                 {
                     connectionFactory.Port = port;
                 }
+
+                ConfigureSsl(connectionFactory, hostName, ssl, insecureSsl);
             }
 
             return connectionFactory;
         }
+
+        internal static void ConfigureSsl(ConnectionFactory connectionFactory, string hostname, bool ssl, bool insecureSsl)
+        {
+            if (ssl)
+            {
+                connectionFactory.Ssl = new SslOption
+                {
+                    Enabled = true,
+
+                    // Set SNI in order to work for multiple RabbitMQ clusters located behind a LoadBalancer
+                    ServerName = hostname,
+                };

```

Comment:
Question: If certificate path is not supplied, wouldn't there be `RemoteCertificateNotAvailable` error?

Reply:
Presenting the certificate without the whole path is not a problem if you set RemoteCertificateChainErrors as an acceptable error. 
Not presenting any certificate will always produce a RabbitMQ.Client.Exceptions.BrokerUnreachableException exception.